src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Services/Implementation/PolicyInformationPoint.cs

```diff
@@ -0,0 +1,117 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Constants;
+using Altinn.Authorization.ABAC.Xacml;
+using Altinn.Platform.Authorization.Configuration;
+using Altinn.Platform.Authorization.Models;
+using Altinn.Platform.Authorization.Repositories.Interface;
+using Altinn.Platform.Authorization.Services.Interface;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Authorization.Services.Implementation
+{
+    /// <summary>
+    /// The Policy Administration Point responsible for storing and modifying delegation policies
+    /// </summary>
+    public class PolicyInformationPoint : IPolicyInformationPoint
+    {
+        private readonly IPolicyRetrievalPoint _prp;
+        private readonly IDelegationMetadataRepository _delegationRepository;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PolicyAdministrationPoint"/> class.
+        /// </summary>
+        /// <param name="policyRetrievalPoint">The policy retrieval point</param>
+        /// <param name="policyRepository">The policy repository</param>
+        /// <param name="delegationRepository">The delegation change repository</param>
+        /// <param name="memoryCache">The cache handler</param>
+        /// <param name="settings">The app settings</param>
+        public PolicyInformationPoint(IPolicyRetrievalPoint policyRetrievalPoint, IPolicyRepository policyRepository, IDelegationMetadataRepository delegationRepository, IMemoryCache memoryCache, IOptions<GeneralSettings> settings)
+        {
```

Comment:
a lot of the properties in the constructor do not seem to be used. e.g. memorycache, generalsettings, policyrepository

Reply:
Removed unused properties.